R/plotting.R

```diff
@@ -87,16 +92,19 @@ names(.all_labs_values) <- .all_aes_names
   brush_set <- !is.null(brush_out$cmd)
 
   # Generating the plot.
-  .create_plot(cmds, se, all.coordinates,
-               param_choices=param_choices, x_lab=x_lab, y_lab=y_lab,
-               color_set=color_set, brush_set=brush_set, color_label=color_label)
+  .create_plot(
+    cmds, se, all.coordinates,
+    param_choices=param_choices, x_lab=x_lab, y_lab=y_lab,
+    color_set=color_set, brush_set=brush_set, color_label=color_label,
+    color_map=color_map, color_name=color_name
+  )
 }
 
 ############################################
 # .make_geneExprPlot  ----
 ############################################
 
-.make_geneExprPlot <- function(se, param_choices, input, all.coordinates)
+.make_geneExprPlot <- function(se, param_choices, input, all.coordinates, color_map, color_name)
```

Comment:
Why does this one need a `color_name` argument, and the others do not? Seems like it gets overridden anyway.

Reply:
leftover of my first implementation, well spotted